z.c

```diff
@@ -1,3 +1,5 @@
+fox edited (edited in progress of PR)
+
 <aaaaaaaaa<<<<<< HEADz
 aaa 
 fox add this
```

Comment:
```suggestion

<aaaaaaaaa<<<<<< HEADz
aaa 
fox add this
```
Should be something like this